pdk/skywater130/build_defs.bzl

```diff
@@ -0,0 +1,103 @@
+"""Skywater 130 PDK support rules.
```

Comment:
What's the rationale of having this file here?

I think the Skywater PDK related files can be under `pdk/skywater130`, or it could be under `dependency_support/...`. But this structure, where there are dependencies from `dependency_support` into the main `rules_hdl` workspace doesn't make sense to me because then there are dependencies from what's supposed to be the dependencies of this repo to the repo itself. It feels very circular to me even though it's technically not.

Reply:
In general I'm a big fan of keeping related (as in providing a certain feature) files in the same place. One reason I'm asking this is because it's not entirely clear what the boundary is that explains why the Skywater PDK should be in two places.